src/styles/indexStyles.js

```diff
@@ -12,3 +12,6 @@ export * from './scenes/ScreenStyles.js'
 export * from './scenes/CryptoExchangeSceneStyles.js'
 export * from './scenes/OtpSettingsScreenStyles.js'
 export * from './scenes/TransactionsExportSceneStyle.js'
+export * from './scenes/OnBoardingSceneStyles.js'
```

Comment:
I like this naming convention and I think we should propose to use it everywhere. Easier to find files.

Reply:
Thanks, thats why I do it. :)